api/app/controllers/review.controller.ts

```diff
@@ -145,17 +152,17 @@ exports.findOne = async (req, res) => {
         message: "Error retrieving review with id=" + id,
       });
     });
-};
+}
 
 // Update a review by the id in the request
-exports.update = async (req, res) => {
+export async function update(req: Request, res: Response) {
   const id = req.params.id;
 
   await Review.update(req.body, {
     where: { id: id },
   })
     .then((num) => {
-      if (num == 1) {
+      if (typeof num === "string" && num == 1) {
```

Comment:
I don't understand this part. Why do we check if num is a `string` and then do an equality check with 1 which is a `number` type

Reply:
Oh, that was a mistake. We should check for number instead